src/Couchbase.Lite.Shared/API/Sync/Replicator.cs

```diff
@@ -183,6 +228,15 @@ public void ResetCheckpoint()
             Config.Options.Reset = true;
         }
 
+        /// <summary>
+        /// Sets replication progress callback level
+        /// </summary>
+        /// <param name="level"> the level to opt in for documents ended and blobs progress callback.</param>
+        public void SetReplicatorOptionProgressLevel(ReplicatorOptionProgressLevel level)
```

Comment:
I don't see this anymore in the spec.  Is it still relevant now that blob progress is removed?

Reply:
removed. Pasin thought its strange to both set the flag and add listener.